### PR TITLE
[project-base] GrapesJS new plugins

### PR DESCRIPTION
**Pull request by TomasGottvald**
#### Description, the reason for the PR
-   added custom table plugin
-   added styleManager
-   added missing translations
-   to change language set - change import of language in initGrapesJs.js file and change language in editor.I18n.setMessages lines - English is imported as default
 
#### Fixes issues
... 

#### Check the appropriate checkboxes below, please

- [ ] [BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/) 
- [x] New feature 
- [x] I have read and signed [License Agreement for contributions](https://www.shopsys.com/license-agreement)

----
:globe_with_meridians: Live Preview:
  - https://tg-ssp-1615-grapejs-new-plugins.odin.shopsys.cloud
  - https://cz.tg-ssp-1615-grapejs-new-plugins.odin.shopsys.cloud

### CODE REVIEW

File: project-base/app/composer.json [hunk 1]
```diff
@@ -157,7 +157,7 @@
             "php app/downloadPhing.php --phing-version=3.0.0": "script",
             "php phing clean": "script",
             "shopsys:domains-urls:configure": "symfony-cmd",
-            "ckeditor:install --clear=skip --release=full --tag=4.5.11": "symfony-cmd"
```
Comment (grossmannmartin) [inline project-base/app/composer.json]: this version requires paid license afaik

Comment (TomasGottvald) [reply project-base/app/composer.json]: But previous used version has been detected with security issue, and recommended this versions. How can I check license usage?

Comment (TomasGottvald) [reply project-base/app/composer.json]: obrazek

Console: This CKEditor 4.5.11 version is not secure. Consider upgrading to the latest one, 4.25.0-lts: https://ckeditor.com/ckeditor-4-support/

Comment (TomasGottvald) [reply project-base/app/composer.json]: Ok according to slack discussion this change will be reverted.

File: project-base/app/assets/js/admin/grapesjs/locales/en.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
+export const cs = {
```
Comment (chlebektomas) [inline project-base/app/assets/js/admin/grapesjs/locales/en.js]: I suppose the const name should be `en`, right?

File: project-base/app/assets/js/admin/grapesjs/locales/en.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+            // 'block-id': 'Block Label',
+        },
+        categories: {
+            // 'category-id': 'Category Label',
+        }
+    },
+    domComponents: {
+        names: {
+            '': 'Box',
+            wrapper: 'Obal',
```
Comment (chlebektomas) [inline project-base/app/assets/js/admin/grapesjs/locales/en.js]: Diff translation

File: project-base/app/translations/messages.en.po [hunk 4]
```diff
@@ -25,6 +25,9 @@ msgstr ""
 msgid "Administrator"
 msgstr ""
 
+msgid "Alt"
+msgstr ""
```
Comment (chlebektomas) [inline project-base/app/translations/messages.en.po:29]: To confirm, is it acceptable to have empty strings? If so, that should be fine.

**Review by chlebektomas — CHANGES_REQUESTED** [review]

File: project-base/app/assets/js/admin/grapesjs/locales/en.js (hunk 3 above)
Comment (TomasGottvald) [reply project-base/app/assets/js/admin/grapesjs/locales/en.js]: Fixed in forgotten commit

File: project-base/app/assets/js/admin/grapesjs/locales/en.js (hunk 2 above)
Comment (TomasGottvald) [reply project-base/app/assets/js/admin/grapesjs/locales/en.js]: Fixed in forgotten commit

File: project-base/app/translations/messages.en.po (hunk 4 above)
Comment (TomasGottvald) [reply project-base/app/translations/messages.en.po:29]: All lines are empty. So I think it is ok

**Review by chlebektomas — APPROVED** [review]

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**